freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/user/model/FmsGroup.java

```diff
@@ -1,20 +1,32 @@
 package com.sequenceiq.freeipa.service.freeipa.user.model;
 
 import java.util.Objects;
+import java.util.Optional;
 
 public class FmsGroup {
 
     private String name;
 
+    private Optional<String> azureObjectId = Optional.empty();
```

Comment:
see comment on UmsUsersStateProvider for why we can't store this info in this model